library/lib/components/menu/ods_dropdown_menu.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+import 'package:ods_flutter/components/menu/item/ods_dropdown_menu_Item.dart';
+
+/// ODS DropdownMenu.
+///
+/// Display a scrolling menu.
+///
+///.
+class OdsDropdownMenu<T> extends StatefulWidget {
+  /// Signature used by PopupMenuButton to lazily construct the items shown when the button is presse
+  final List<OdsDropdownMenuItem<String>> items;
+
+  /// Called when the user selects a value from the popup menu created by this button
+  final Function(String)? onClick;
```

Comment:
why onClick her and 
selectedItem on OdsExposedDropdownMenu
should be similar ?